packages/confirm-dialog/src/vaadin-lit-confirm-dialog.js

```diff
@@ -68,11 +68,9 @@ class ConfirmDialog extends ConfirmDialogMixin(ElementMixin(ThemePropertyMixin(P
   }
 
   /** @protected */
-  async ready() {
+  ready() {
```

Comment:
BTW, the `ready()` block is now the same in Polymer and Lit versions so it could be moved to the mixin.

Reply:
Thanks, done.